awesomeConfig.cmake

```diff
@@ -4,7 +4,7 @@ set(PROJECT_AWE_NAME awesome)
 # `git describe` later.
 set(VERSION devel)
 
-set(CODENAME "The Fox")
+set(CODENAME "Harder, Better, Faster, Stronger")
```

Comment:
I would have liked an RC if it was released 2 weeks ago, but  it was more of a marketing move than anything else. I did a the marketing anyway using "soon" instead of "RC". It produced the few bug reports we had, pointed to flaws in the doc and got us a new screenshot, so I am happy.

In the rolling scenario, "Harder, Better, Faster, Stronger" would be the codename for Awesome 4.